crates/subspace-farmer/src/bin/subspace-farmer/commands/farm.rs

```diff
@@ -58,7 +56,7 @@ pub(crate) async fn farm_multi_disk(
 
     let FarmingArgs {
         bootstrap_nodes,
-        listen_on,
+        listen_on: _,
```

Comment:
Hm, I would have expected for `listen_on` to still be wired to networking instances, now it is hanging in the air.

Reply:
Yes, it is. I'm not sure we know the 'integration point' between the new consensus and the networking layer now. I removed obsolete features but kept CLI arguments for now because args could be reused in contrast with other code.

Reply:
I don't see how integration impacts it, just send `listen_on` down the stack. If port is already used, networking should use a random port anyway.

Reply:
We will use the listen-on CLI argument but I'm not sure where exactly will we host the networking instance (what module?). It seems that we'll know it after the new plotting design. This is why I kept it but removed other related code.

Reply:
But we still have networking instances in the farmer, they still have `listen_on` in their configuration, but thing value is ignored instead of being sent down the stack. That is the only reason I commented on it: there is a clear path for this value to be used. It is kind of irrelevant as to what networking will look like in the future.